projects/igniteui-angular/schematics/ng-add/index.ts

```diff
@@ -2,86 +2,26 @@ import { workspaces } from '@angular-devkit/core';
 import { chain, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
```

Comment:
Can you also remove the prompt setting for the ng-add schematic in the schema.json file?